HelpMyStreetFE/HelpMyStreetFE/Controllers/RequestHelpController.cs

```diff
@@ -194,6 +194,10 @@ public async Task<IActionResult> RequestHelp(string referringGroup, string sourc
             if (requestHelpJourney.AccessRestrictedByRole)
             {
                 var user = await _authService.GetCurrentUser(HttpContext, cancellationToken);
+                if (user == null)
+                {
+                    return RedirectToAction("403", "Error");
+                }
                 var userHasPermission = await _groupMemberService.GetUserHasRole(user.ID, referringGroupId, GroupRoles.RequestSubmitter, cancellationToken);
```

Comment:
This could be more compact with 
`var userHasPermission = user != null && await _groupMemberService.GetUserHasRole(user.ID, referringGroupId, GroupRoles.RequestSubmitter, cancellationToken);`

Reply:
(The part after && will only be evaluated if the part before checks out.)